src/source/ObjcppGenerator.scala

```diff
@@ -138,6 +138,10 @@ class ObjcppGenerator(spec: Spec) extends BaseObjcGenerator(spec) {
       val objcSelf = if (i.ext.objc && i.ext.cpp) self + "CppProxy" else self
 
       if (i.ext.cpp) {
+		// Disable warnings in Clang about directly accessing the ivars of an object
+		w.wl("#if __clang__")
```

Comment:
Looks like the indentation of the new code here is mixing tabs and spaces.  Can you standardize on spaces to match the existing code, please?